hazelcast/src/main/java/com/hazelcast/internal/partition/operation/PartitionReplicaSyncRequestOffloadable.java

```diff
@@ -174,4 +219,35 @@ protected void readInternal(ObjectDataInput in) throws IOException {
         namespaces.addAll(readCollection(in));
         partitionId = in.readInt();
     }
+
+    static final class PartitionNamespaceTuple {
```

Comment:
Maybe we can create a generic tuple class instead and move it to `com.hazelcast.internal.util` where we already have `TriTuple<X, Y, Z>` class

Reply:
Good suggestion, I removed the redundant tuple class in favour of existing `BiTuple`.